autogluon/utils/tabular/features/generators/rename.py

```diff
@@ -0,0 +1,72 @@
+import copy
+import logging
+
+from pandas import DataFrame
+
+from .abstract import AbstractFeatureGenerator
+
+logger = logging.getLogger(__name__)
+
+
+class RenameFeatureGenerator(AbstractFeatureGenerator):
+    """
+    RenameFeatureGenerator renames the columns without altering the features.
```

Comment:
"altering the features" -> "altering their values"?

Reply:
maybe clarify to users why column names might matter?

Reply:
clarified